activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -248,11 +248,23 @@ def add_column(table_name, column_name, type, **options) #:nodoc:
       end
 
       def remove_column(table_name, column_name, type = nil, **options) #:nodoc:
-        alter_table(table_name) do |definition|
-          definition.remove_column column_name
-          definition.foreign_keys.delete_if do |_, fk_options|
-            fk_options[:column] == column_name.to_s
+        old_way = ->() {
```

Comment:
I'm not sure this is the _correct_ way to get this change into Rails, but I can't offer a better suggestion atm 🤔